src/Helper/Base64UrlConverter.php

```diff
@@ -1,33 +1,76 @@
 <?php namespace ReallySimpleJWT\Helper;
 
+/**
+ * A class that converts Base64 strings to Base64Url strings and vice versa
+ * for JWT Tokens
+ *
+ * @author Rob Waller <rdwaller1984@gmail.com>
+ */
 class Base64UrlConverter
 {
+    /**
+     * The base64 string
+     *
+     * @var string
+     */
     private $base64String;
 
+    /**
+     * The base64 Url string
+     *
+     * @var string
+     */
     private $base64UrlString;
 
-    public function setBase64String($base64String)
+    /**
+     * Set the base64 string first step in
+     *
+     * @param string $base64String
+     *
+     * @return self
+     */
+    public function setBase64String(string $base64String): self
     {
         $this->base64String = $base64String;
 
         return $this;
     }
 
-    public function setBase64UrlString($base64UrlString)
+    /**
+     * Set the base64 Url string
+     *
+     * @param string $base64UrlString
+     *
+     * @return self
+     */
+    public function setBase64UrlString($base64UrlString): self
```

Comment:
Possibly typehint the parameter as a string?

Reply:
Good spot, I will change.